deploy/helm/secret-operator/README.md

```diff
@@ -0,0 +1,33 @@
+# Helm Chart for Stackable Operator for Stackable Secret Operator
+
+This Helm Chart can be used to install Custom Resource Definitions and the Operator for Stackable Secret Operator provided by Stackable.
```

Comment:
Ok i dont think that naming makes sense, but its consistent :D
`This Helm Chart can be used to install Custom Resource Definitions and the Secret Operator provided by Stackable.`